docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,80 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there have been three versions for how Mobile UCRs were sent to mobile devices:
+
+- **Version 1.0**: All reports are bundled together into a single fixture during the restore process.
+- **Version 1.5**: Both V1 and V2 are supported together. This is used to migrate from V1 without causing any breaking changes.
+- **Version 2.0**: Enhanced performance where each report has a fixture of its own.
+
+Deprecation of Mobile UCR V1 and V1.5
+-------------------------------------
+Both V1 and V1.5 have been deprecated to address the performance challenges encountered with these versions.
+
+Benefits of Upgrading to Mobile UCR V2
+--------------------------------------
+The introduction of Mobile UCR V2 offers significant performance and efficiency improvements around how UCR data is restored and synced.
+These improvements offer faster report lookups and restores, as well as a new sync delay option for each report that allows administrators to control
+how frequently report data is synced.
+
+Moving to Mobile UCR V2
+-----------------------
+- Applications that do not have any references to UCRs in forms, case lists and filters need only have their
+  Mobile UCR Version updated to V2 and the app will continue working as normal.
+- Applications with references to reports will need to have all manual references to the `commcare:reports`
```

Comment:
can we add what the format was in V1?
So, this would be 
"references from commcare-reports to commcare:reports..."
So, users don't need to figure out what to look for?

Reply:
> can we add what the format was in V1?

This will be different depending on whether it is a report, filter, or column reference. Examples of these formats are given in the "Update Application Manual References", so I'll add a note here to refer to that section for more details.

Reply:
I've added a reference, along with a clear example in [da3846a](https://github.com/dimagi/commcare-hq/pull/35276/commits/da3846ae0de7d0591b6923b014f4f5b1d4ffee60).